src/theme/ThemeTypography.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
is it autogenerated?

Reply:
This file isn't auto generated

Reply:
then maybe better to use variables for colors, heights? Or the numbers is completely random and can't be counted from some base value?